app/views/bookings/request_refund.js.erb

```diff
@@ -0,0 +1,4 @@
+location.reload();
+<% if flash[:notice].present? %>
+  Materialize.toast("<%= flash[:notice].to_s %>", 3000, "rounded")
```

Comment:
I think we need to discuss on having just one `Materialize.toast()` method in our entire application.